### PR TITLE
Api

### PR DESCRIPTION
**Pull request by JoseLopez235**
PR Contain:
- All API calls to the movie websites
- Movie object class that simplifies information
- VCR/Webmock
- All Passing Test

### CODE REVIEW

File: app/poros/movie_object.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+class MovieObject
```
Comment (GDemps) [inline app/poros/movie_object.rb:1]: Great work on this class and breaking it down into methods.

File: app/poros/movie_object.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        if hash[name]
+          hash[name] << ",#{cast[:character]}"
+        else
+          hash[name] = "#{cast[:character]}"
+        end
+      end
+    end
+  end
+
+  def detail_runtime
```
Comment (SageOfCode) [inline app/poros/movie_object.rb:37]: Great work on this method!

File: app/service/movie_api_service.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    end
+    json_response = JSON.parse(response.body, symbolize_names: true)
+    MovieObject.new(json_response)
+  end
+
+  def self.top_rated_movies
+    page = 1
+    movies = []
+    2.times do
+      response = conn.get("/3/movie/top_rated") do |req|
```
Comment (SageOfCode) [inline app/service/movie_api_service.rb:17]: Could this response be refactored into a helper method?

**Review by SageOfCode — COMMENTED** [review]
Looks good to me, great work here.

**Pull request merged**